### ISSUE TITLE
Heart Icon Not Showing, Just Showing Square

### ISSUE THREAD

**Issue by ChildeRoland840** [algoritmika/wish-list-for-woocommerce#22]
After updating every plugin, woo and wp, the heart icon is just showing the dreaded "empty square" and as I try to pinpoint and figure the the problem, I don't see anything wrong other than this site is still using FA 4.4, not 5.

**Comment by pablo-sg-pacheco** [comment]
Hi @ChildeRoland840 ,
If that's the case, please try to update your **FontAwesome URL** option to https://use.fontawesome.com/releases/v5.5.0/css/all.css

**Comment by ChildeRoland840** [comment]
Actually, I ended up having to go into the templates and add fa to the
beginning of the class for the icon.  I did try using fa 5.5.0 but that
didn't fix the issue because the icon was missing the fa at the beginning
of the class.  It just had "fa-heart" instead of "fa fa-heart".

Thanks,

Ted Hust
Project Manager
Aarc Media Group LLC
thust@aarcmediagroup.com
http://www.aarcmediagroup.com
Phone:  646-685-3563

*** Celebrating 14 Years of Outstanding Service & Support ***

**Comment by pablo-sg-pacheco** [comment]
Hum,
Are you using the most recent version (1.6.8)? I'm pretty sure I had updated the font-awesome classes.

Actually, now it is like:
`fas fa-heart`

**Comment by ChildeRoland840** [comment]
Yeah, we updated to your latest version.  What I did notice was that in the
actual "rendered source" it was only showing fa-heart and not the fa
fa-heart nor the fas fa-heart.  I purposely made the update to the plugin
core files figuring I'd see what happened the next time you put out an
update.  For now, we are good and my fix is working, I just figured I'd let
you know.  The module has worked well for the client for pushing 2 years
now so I thank everyone involved for developing.

Thanks,

Ted Hust
Project Manager
Aarc Media Group LLC
thust@aarcmediagroup.com
http://www.aarcmediagroup.com
Phone:  646-685-3563

*** Celebrating 14 Years of Outstanding Service & Support ***

**Comment by pablo-sg-pacheco** [comment]
Great!
Thanks for the kind words :)

Anyway, I will look into it again. I really don't know why it's happening for you.
Thanks for letting me know ;)

**Comment by ChildeRoland840** [comment]
DON'T LOOK TOO HARD haha.  As stated, I did try using FA5, and that didn't
update the class to include the "fas" as you stated, so it still didn't
work for me in that scenario, so I switched back to FA4, and simply added
the "fa" myself and that was working.  So it is still possible the issue is
isolated to my install and I don't want to cause you to stare at code for a
fix that maybe doesn't need fixing haha.

Thanks,

Ted Hust
Project Manager
Aarc Media Group LLC
thust@aarcmediagroup.com
http://www.aarcmediagroup.com
Phone:  646-685-3563

*** Celebrating 14 Years of Outstanding Service & Support ***

**Comment by pablo-sg-pacheco** [comment]
Ok ;)
